.github/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+package-lock.json
```

Comment:
```suggestion
```

If your workflow has dependencies, unfortunately you'll need to commit (and keep updated) a `package-lock.json`.